tests/views/test.del-pay-method-confirmation.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import TestUtils from 'react/lib/ReactTestUtils';
+
+import $ from 'jquery';
+import ConfirmDelPayMethod from 'views/management/confirm-del-pay-method';
+
+import * as helpers from '../helpers';
+
+describe('Confirm Delete Pay Methods', function() {
+
+  beforeEach(function() {
+
+    this.fakePayMethods = [
+      {
+        'id': 1,
+        'resource_uri': '/pay-method/1/',
+        'truncated_id': '4321',
+        'type_name': 'visa',
+      }, {
+        'id': 2,
+        'resource_uri': '/pay-method/2/',
+        'truncated_id': '1234',
+        'type_name': 'amex',
+      }, {
+        'id': 23,
+        'resource_uri': '/pay-method/3/',
+        'truncated_id': '4321',
+        'type_name': 'mastercard',
+      },
+    ];
+
+    this.fakeSubs = [
+      {
+        'id': 1,
+        'paymethod': '/pay-method/2/',
+        'resource_uri': '/sub/1/',
+        'seller_product': {
+          'external_id': 'mozilla-concrete-mortar',
+          'public_id': 'mozilla-concrete-mortar',
+          'resource_pk': 1,
+          'resource_uri': '/generic/product/1/',
+        },
+      }, {
+        'id': 2,
+        'paymethod': '/pay-method/2/',
+        'resource_uri': '/sub/2/',
+        'seller_product': {
+          'external_id': 'mozilla-concrete-brick',
+          'public_id': 'mozilla-concrete-brick',
+          'resource_pk': 2,
+          'resource_uri': '/generic/product/2/',
+        },
+      },
+    ];
+
+    this.delPayMethodStub = sinon.stub();
+    this.delPayMethodStub.returns($.Deferred().resolve());
+
+    this.getSubsByPayMethodStub = sinon.stub();
+    this.showDelPayMethodStub = sinon.stub();
+    this.showPayMethodsStub = sinon.stub();
+    this.updateSubPayMethodStub = sinon.stub();
+
+    this.mountView = function(payMethodUri,
```

Comment:
I think it makes the calling code a lot more flexible and easier to read when you use keyword args like:
````
this.mountView = function({payMethodUri, 
                           affectedSubscriptions=this.fakeSubs,
                           payMethods=this.fakePayMethods})
this.mountView({payMethodUri: '/pay-method/2/'})
````